### PR TITLE
feat: add rem

### PR DESCRIPTION
**Pull request by MiaoRunZhuang**

### CODE REVIEW

**Review by zhangjing28 — APPROVED** [review]

**Review by Chaoming-L — APPROVED** [review]

File: src/utils/rem.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+export enum DeviceSize {
+    PC = 1440,
+    WAP = 393,
```
Comment (jasonintju) [inline src/utils/rem.ts:3]: 390?

File: src/utils/rem.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * @description Get the rem ratio.
+ */
+export const getRemRatio = () => {
+    return 1 / 16;
+};
+
+/**
+ * @description Convert px to rem.
+ */
+export const px2rem = (value: number, hasUnit = true) => {
```
Comment (jasonintju) [inline src/utils/rem.ts:32]: Where is this function applied?

Comment (MiaoRunZhuang) [reply src/utils/rem.ts:32]: The `postcss-pxtorem` plugin operates at the compilation stage and can only compile static classes or CSS. Sometimes, if you need to calculate or dynamically set a px value in JavaScript, you'll need to convert it to rem manually.

File: src/utils/rem.ts (hunk 1 above)
Comment (MiaoRunZhuang) [reply src/utils/rem.ts]: done

**Pull request merged**